osu.Game/Graphics/Containers/Markdown/OsuMarkdownContainer.cs

```diff
@@ -11,14 +11,19 @@
 using Markdig.Extensions.Tables;
 using Markdig.Extensions.Yaml;
 using Markdig.Syntax;
+using osu.Framework.Allocation;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
 using osu.Framework.Graphics.Containers.Markdown;
+using osu.Framework.Graphics.Containers.Markdown.Footnotes;
 using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics.Containers.Markdown.Footnotes;
 using osu.Game.Graphics.Sprites;
+using osuTK;
 
 namespace osu.Game.Graphics.Containers.Markdown
 {
+    [Cached]
```

Comment:
Cached for two purposes: being able to pull out a fragment of the markdown document for the tooltip, as well for the sake of document navigation.